event.php

```diff
@@ -1228,22 +1228,27 @@ function updateEvent()
     $event->metaurl = $_POST['metaurl'];
     $event->reporturl = $_POST['reporturl'];
 
-    if ($_POST['mainrounds'] == '') {
-        $_POST['mainrounds'] = 3;
-    }
-    if ($_POST['mainstruct'] == '') {
-        $_POST['mainstruct'] = 'Swiss';
-    }
-    $event->mainrounds = $_POST['mainrounds'];
-    $event->mainstruct = $_POST['mainstruct'];
-    if ($_POST['finalrounds'] == '') {
-        $_POST['finalrounds'] = 0;
-    }
-    if ($_POST['finalstruct'] == '') {
-        $_POST['finalstruct'] = 'Single Elimination';
+    if (!$event->inSingleEliminationRound()) {
```

Comment:
This doesn't do what you want it to do.  

Unless you want it to prevent people from ever modifying a tournament that starts with a Single-Elim portion? (You don't)

Reply:
well I have remove it for now. I still think changing number of rounds in single elim portion is bad since the pairing code would produce weird pairing